app/main.development.js

```diff
@@ -503,15 +521,23 @@ const launchDCRWallet = () => {
     }
   });
 
-  if (debug) {
-    dcrwallet.stdout.on("data", (data) => {
-      process.stdout.write(`${data}`);
-    });
+  dcrwallet.stdout.on("data", (data) => {
+    if (dcrwalletLogs) {
+      dcrwalletLogs = Buffer.concat([dcrwalletLogs, Buffer.from(data)]);
```

Comment:
Should we worry about adding a maximum size for the `dcrwalletLogs` buffer here? Otherwise this will just keep getting bigger and on a long lived wallet may start to cause problems...

Reply:
Also, a stylistic (optional) note, just cause I can't help it ;P

I'd initialize `dcrwalletLogs` as an empty buffer to avoid the `if` here, if that var is never nulled.

Reply:
Agree on the max size, just not sure how best to implement.  If greater than max then pop the front of the current dcrwalletLogs buffer and then continue with concat of new data?